library.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
```

Comment:
Please improve the documentation/commenting of the function. The return value is not clear, the described functionality (`prints`) is different from the actual function (`print` and `return str`).